api/swagger/forms.yaml

```diff
@@ -312,11 +373,13 @@ components:
         type: string
     locale:
       name: locale
-      in: query
-      description: form locale
+      in: path
+      description: form locale represented as language tag.
+      required: true
```

Comment:
as per open api path params cannot be optional. we so had to make it mandatory. But for us, it is optional.
the way to support this would be to have an additional path with locale in path param.